src/attributes/attr.js

```diff
@@ -34,7 +34,7 @@ jQuery.extend( {
 		}
 
 		// Fallback to prop when attributes are not supported
-		if ( typeof elem.getAttribute === "undefined" ) {
+		if ( elem.getAttribute === undefined ) {
```

Comment:
This was [intentional](https://github.com/mixed/jquery/commit/075da3091cda170bd8cd5ce47bad1c5b14760232), but probably not needed anymore. @jquery/core Does anyone remember at what point IE stopped automatically invoking the method on mere reference?